environments/common/inventory/groups

```diff
@@ -26,6 +26,10 @@ login
 openhpc
 additional
 
+[journald:childen]
```

Comment:
If its non-empty here, rather than environments/site/inventory/groups, it can NEVER be turned off. Is that OK?

Reply:
I wouldn't expect anyone to want to turn this off. You can still tune journald_storage to volatile, but I'm not sure why you'd want to do that.

Reply:
I've moved it to environments/site as that sounded preferable